### DISCUSSION TITLE
Does this extension support autocomplete through JS file?

### DISCUSSION THREAD

**Discussion by tej-mosaic** [vunguyentuan/vscode-css-variables#92] [Q&A]
I have my theme defined in a javascript file with at `.js` extension. I added if in the lookupFiles key in the `settings.json`, but the autocomplete was not working. Just wanted to confirm if this extension supports js file for lookup?

**Comment by lmn451** [comment]
@vunguyentuan same for and html files, I think we can think of creating css lsp. What do you think?
